block_test.go

```diff
@@ -150,16 +157,15 @@ func TestCorruptedChunk(t *testing.T) {
 }
 
 // createBlock creates a block with given set of series and returns its dir.
-func createBlock(tb testing.TB, dir string, series []Series) string {
+func createBlock(tb testing.TB, dir string, series []Series) (string, error) {
```

Comment:
looking at the changes you don't need to refactor this anymore.

Reply:
I meant it can stay as before:
`func createBlock(tb testing.TB, dir string, series []Series) string {`

Reply:
> I meant it can stay as before:
> `func createBlock(tb testing.TB, dir string, series []Series) string {`
> 
> if it is not needed in this PR can refactor it when needed.

Restore it as before